/**:
  ros__parameters:
    dist_threshold: 15.0
